recipes/app3.rb

```diff
@@ -138,3 +134,52 @@
   ]
   sensitive true
 end
+
+# Check if attribute is set for testing
+db_host = node['osl-app'].attribute?('db_hostname') ? node['osl-app']['db_hostname'] : creds['db_hostname']
+
+etherpad_tag = '1.8.6-2020.11.13.2015'
+etherpad_sd_tag = '1.8.6-2020.11.13.2015'
```

Comment:
I think it's short enough that you can give this a more descriptive name. Would you mind doing that for this and `etherpad_sd_creds` below?
```suggestion
etherpad_snowdrift_tag = '1.8.6-2020.11.13.2015'
```